tchannel/tchannel.py

```diff
@@ -268,7 +268,7 @@ def advertise(self, routers, name=None, timeout=None):
             other services will use to make contact with you.
 
         :param timeout:
-            The timeout (in seconds) for the initial advertise attempt.
+            The timeout (in sec) for the initial advertise attempt.
```

Comment:
+1 keep as sec. As mentioned in the other PR also confirm that advertise correctly passes 30000 timeout.